docs/automate.md

```diff
@@ -0,0 +1,57 @@
+# Automate Configuring Self-Hosted Runners
+
+
+## Export PAT
+
+Before running any of these sample scripts, create a GitHub PAT and export it before running the script
+
+```bash
+export RUNNER_CFG_PAT=yourPAT
+```
+
+## Create running as a service
+
+**Scenario**: Run on a machine or VM (not container) which automates:
+
+ - Resolving latest released runner
+ - Download and extract latest
+ - Acquire a registration token
+ - Configure the runner
+ - Run as a systemd (linux) or Launchd (osx) service
+
+:point_right: [Sample script here](../scripts/create-latest-svc.sh) :point_left:
+
+Run as a one-line. NOTE: replace with yourorg/yourrepo (repo level) or just yourorg (org level) 
+```bash
+curl -s https://raw.githubusercontent.com/actions/runner/automate/scripts/create-latest-svc.sh | bash -s yourorg/yourrepo
+```
+
+## Uninstall running as service 
+
+**Scenario**: Run on a machine or VM (not container) which automates:
+
+ - Stops and uninstalls the systemd (linux) or Launchd (osx) service
+ - Acquires a removal token
+ - Removes the runner
+
+:point_right: [Sample script here](../scripts/remove-svc.sh) :point_left:
+
+Repo level one liner.  NOTE: replace with yourorg/yourrepo (repo level) or just yourorg (org level) 
+```bash
+curl -s https://raw.githubusercontent.com/actions/runner/automate/scripts/remove-svc.sh | bash -s yourorg/yourrepo
+```
+
+### Delete an offline runner
```

Comment:
do we need this? we are going to clean up dormant runners if the runner offline for 30 days.

Reply:
Users might not want that offline runner showing up in the settings UI for 30 days

Reply:
my question is more like when should we expect the customer to use this?
I can imagine in an autoscale VM/Kubernetes scenario, the customer will use the config script to set up a runner when creating VM/container and use the un-config script to un-config runner before we delete the VM/container.
If the VM/container crash or shutdown unexpected, we will use the delete offline script for cleanup. 

@bryanmacfarlane is my understand right?

Reply:
It's used by the same people that need the delete api (as opposed to the cli to remove).  This is used for outside/in automation where the runner goes away.  The customers I've interacted with so far for org level runners were using it for container / K8s runners where there's just an entry point.  The container is stopped outside in and then they to delete it after it goes away.